src/renderer/hooks/useTestImport.ts

```diff
@@ -55,7 +62,12 @@ export const useTestImport = () => {
         error: null,
       };
     } else {
-      const res = await discoverPytest(path, settings.importAsOneRef);
+      let res: Result<TestDiscoverContainer, Error>;
+      if (settings.importType === "pytest") {
```

Comment:
should probably use `ts-pattern` to exhaustively match so when we add another type it will error

Reply:
Oh this is nice :O